Game cycle and Cinemachine

Chumakov123 opened the pull request:
Завершена работа над паттерном игрового цикла (Меню -> Вступление -> Игра -> Концовка -> Титры). Исправлено отображение параллакса. Добавлен пропуск вступительной катсцены по клавише [Enter]. Добавлена концовка и переход на сцену с титрами. Добавлена камера Cinemachine на все сцены, скорректированы скрипты под работу Cinemachine. Добавлен TrailRenderer для снарядов

Chumakov123 committed:
Game cycle

Завершена работа над паттерном игрового цикла (Меню -> Вступление -> Игра -> Концовка -> Титры). Исправлено отображение параллакса. Добавлен пропуск вступительной катсцены по клавише [Enter]. Добавлена концовка и переход на сцену с титрами

EnderGolem commented:
Я не нашел баг, который появился у него с ногами в редакторе. Но если появится у всех надо ревертать

Chumakov123 committed:
Add Cinemachine

Добавлена камера Cinemachine на все сцены, скорректированы скрипты под работу Cinemachine. Добавлен TrailRenderer для снарядов
